htdocs/core/class/commonobject.class.php

```diff
@@ -6152,6 +6152,10 @@ function showOptionals($extrafields, $mode='view', $params=null, $keysuffix='',
 			$e = 0;
 			foreach($extrafields->attributes[$this->table_element]['label'] as $key=>$label)
 			{
+			    if(empty($extrafields->attribute_list[$key]) && $mode == 'view'){
```

Comment:
$extrafields->attribute_xxx are deprecated vars.
Only $extrafields->attributes[$this->table_element][...] must be used.